RF24.h

```diff
@@ -1277,11 +1277,11 @@ class RF24
      * @code
      * bool goodSignal = radio.testRPD();
      * if(radio.available()){
-     *    Serial.println(goodSignal ? "Strong signal > 64dBm" : "Weak signal < 64dBm" );
+     *    Serial.println(goodSignal ? "Strong signal > -64dBm" : "Weak or no signal < -64dBm" );
      *    radio.read(0,0);
```

Comment:
I think this flushes 1 level of RX FIFO (if not a non-op under the hood). Maybe instead use
```suggestion
     *    radio.flush_rx();
```
 Alternatively, and kinda hacky, I think `radio.read(0, 96)` would flush the entire RX FIFO and clear the "data ready" IRQ flag.

Reply:
It is supposed to only flush one level of the FIFO. Its an example of how to measure the RPD of received payloads. Maybe it should be changed to `radio.read(&payload, sizeof(payload));` so its clearer?

Reply:
On second thought, it should just read "weak signal' because if available returns true there was a payload received regardless of RPD.